### ISSUE TITLE
Update smoothState.js latest release within GitHub

### ISSUE THREAD

**Issue by olivercastle** [miguel-perez/smoothState.js#226]
@miguel-perez Any chance you can update the latest release for smoothState within GitHub. Downloaded the 0.5.2v by mistake, spent a bit of time figuring out why nothing was working to the docs.

Thanks.

**Comment by olivercastle** [comment]
This also might help solve #191

**Comment by JohnnyWalkerDigital** [comment]
I can't even find any version numbers? How do you know you downloaded 0.5.2?

**Comment by olivercastle** [comment]
They are all listed under the releases:

https://github.com/miguel-perez/smoothState.js/releases

But the latest release is tagged on old version.

**Comment by JohnnyWalkerDigital** [comment]
Thanks! 

Very confusing, though. Which one is the latest? Can't I just grab it from master? Or do I need to get 0.7.2 from releases? I really wish there was a @version for each of them.

I wish Miguel would/could spend more time on this project. It has the possibility of really taking off and becoming something big IMO, but it's always a battle at the moment. I used it several months ago, while Miguel was waiting for permission to take control of it from his old employer, and I had hoped that all the kinks had been ironed out by now.

**Comment by olivercastle** [comment]
Personally i took the 0.7.2 release. The master should be fine too.

Yes, its a great project. I've recently added it to my site and it works wonders.. Had to do a little bit of work as it was running on Drupal 8 install.

http://www.fullbundle.com

**Comment by JohnnyWalkerDigital** [comment]
Yes, it really has potential to change things. I've used it a few times now, for example here:

http://www.walkforwards.co.uk/

**Comment by JohnnyWalkerDigital** [comment]
Hmm. There's been 56 commits to master since 0.7.2 was tagged. I think you should really just be using the master branch and ignoring the releases.

**Comment by olivercastle** [comment]
Wow. Yes, i think your right. Might have to update mine. Not sure if i did already.

**Comment by JohnnyWalkerDigital** [comment]
Phew. Things aren't quite as confusing as I'd feared then!
